fuzz/src/chanmon_consistency.rs

```diff
@@ -702,78 +702,76 @@ pub fn do_test<Out: test_logger::Output>(data: &[u8], out: Out) {
 			0x00 => *monitor_a.update_ret.lock().unwrap() = Err(ChannelMonitorUpdateErr::TemporaryFailure),
 			0x01 => *monitor_b.update_ret.lock().unwrap() = Err(ChannelMonitorUpdateErr::TemporaryFailure),
 			0x02 => *monitor_c.update_ret.lock().unwrap() = Err(ChannelMonitorUpdateErr::TemporaryFailure),
-			0x03 => *monitor_a.update_ret.lock().unwrap() = Ok(()),
-			0x04 => *monitor_b.update_ret.lock().unwrap() = Ok(()),
-			0x05 => *monitor_c.update_ret.lock().unwrap() = Ok(()),
-			0x06 => {
+			0x04 => *monitor_a.update_ret.lock().unwrap() = Ok(()),
+			0x05 => *monitor_b.update_ret.lock().unwrap() = Ok(()),
+			0x06 => *monitor_c.update_ret.lock().unwrap() = Ok(()),
+
+			0x08 => {
```

Comment:
Could you help me understand why values like `0x03` and `0x07` are skipped?

Reply:
To keep related message groups having the same binary prefix. Honestly it probably doesn't matter, but fuzzers do do some bit-twiddling-based tweaking and I figured no reason to not have bit-twiddling often have similar effects. I noted it in a new comment.